src/os/bluestore/BlueFS.cc

```diff
@@ -842,6 +841,7 @@ int BlueFS::_replay(bool noop, bool to_stdout)
         {
 	  bluefs_fnode_t fnode;
 	  ::decode(fnode, p);
+	  fnode.recalc_allocated();
```

Comment:
@ifed01 i think this is a bug fix and should be backported to luminous. am i right?